webapp/pages/post/_id/_slug/index.vue

```diff
@@ -1,6 +1,7 @@
 <template>
   <transition name="fade" appear>
     <ds-card
+      :lang="post.language"
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-11-14T23:52:57Z" title="Friday, November 15th 2019, 12:52:57 am +01:00">Nov 15, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,6 +1,7 @@
 <template>
   <transition name="fade" appear>
     <ds-card
+      :lang="language"
```


```suggestion
      :lang="post.language"
```